### PR TITLE
rosdep/python: add python-pytest-qt

### PR DESCRIPTION
**Pull request by machinekoder**
Add the [pytest-qt](https://pypi.org/project/pytest-qt/) pytest extension.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2672,6 +2672,25 @@ python-pytest:
   fedora: [python-pytest]
   gentoo: [dev-python/pytest]
   ubuntu: [python-pytest]
+python-pytest-qt:
+  debian:
+    buster: [pytest-qt]
+    jessie:
+      pip:
+        [pytest-qt]
+    wheezy:
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: this is out of alphabetical order

File: rosdep/python.yaml [hunk 2]
```diff
@@ -2672,6 +2672,25 @@ python-pytest:
   fedora: [python-pytest]
   gentoo: [dev-python/pytest]
   ubuntu: [python-pytest]
+python-pytest-qt:
+  debian:
+    buster: [pytest-qt]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: there is no binary deb available on buster yet.

**Review by mikaelarguedas — CHANGES_REQUESTED** [review]
This package doesn't have binary packages available for our targeted platforms (Ubuntu and Debian).
So users cannot release packages depending on them

Can you please make this key a pip key (with the suffix `-pip`) ?

File: rosdep/python.yaml (hunk 2 above)
Comment (machinekoder) [reply rosdep/python.yaml]: You are right. There is one for python3 though: https://packages.debian.org/buster/python3-pytestqt

File: rosdep/python.yaml [hunk 3]
```diff
@@ -2672,6 +2672,18 @@ python-pytest:
   fedora: [python-pytest]
   gentoo: [dev-python/pytest]
   ubuntu: [python-pytest]
+python-pytest-qt:
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: We can remove that key altogether as it doesn't have any resolution for the platforms we require support for.

We will consider adding it back once it's available for all platforms targeted by at least one ROS distribution

Comment (machinekoder) [reply rosdep/python.yaml]: sure

File: rosdep/python.yaml (hunk 2 above)
Comment (mikaelarguedas) [reply rosdep/python.yaml]: > There is one for python3 though: https://packages.debian.org/buster/python3-pytestqt

:+1:
Currently there is no ROS distribution targeting Python3. Though that will likely change in the future https://github.com/ros-infrastructure/rep/pull/149

**Review by mikaelarguedas — APPROVED** [review]
lgtm

**Pull request merged**